conformance/lang/expressions/error_constructor.balt

```diff
@@ -0,0 +1,700 @@
+Test-Case: parser-error
+Description: Test error constructor negative syntax.
+Labels: error-constructor-expr, error-type, module-type-defn, string
+
+type E error;
+
+function errorFunction() {
+    error err1 = error; // @error missing open and close parenthesis tokens
+    error err2 = error(; // @error missing close parenthesis token
+    error err3 = error); // @error missing open parenthesis token
+    error err4 = error(); // @error missing argument within parenthesis
+    error err5 = error E; // @error missing open and close parenthesis tokens
+    error err6 = error E(; // @error missing close parenthesis token
+    error err7 = error E); // @error missing open parenthesis token
+    error err8 = error E(); // @error missing argument within parenthesis
+    error err9 = error("error message" ()); // @error missing comma token
```

Comment:
let's add some more missign comma tests like missing comma before named argument and also add more tests with invalid tokens inside error constructor expression.